sorc/rtma_post.fd/sorc/ncep_post.fd/CALHEL3.f

```diff
@@ -373,7 +373,7 @@ SUBROUTINE CALHEL3(LLOW,LUPP,UST,VST,HELI)
 !       COMPUTE STORM-RELATIVE HELICITY
 !
 !!$omp  parallel do private(i,j,n,l,du1,du2,dv1,dv2,dz,dz1,dz2,dzabv,ie,iw,jn,js,z1,z2,z3)
-      DO N=1,2 ! for dfferent helicity depth
+!      DO N=1,2 ! for dfferent helicity depth
```

Comment:
Hi @EdwardColon-NOAA,

I am not familiar with the UPP code. so I just would like to confirm with you that, this modification (commenting off the line 376  "DO N=1,2) is it a fix to the bug of the weird behavior of Helicity and Wind Shear in the 3DRTMA web portal of HWT website you showed to us in the meeting today? Is the modification also used in regular UPP for RAP/HRRR system? Or it is specially used for out 3DRTMA UPP? 

Thank you!

-Gang